cli/src/semgrep/meta.py

```diff
@@ -22,6 +22,35 @@
 logger = getLogger(__name__)
 
 
+def get_url_from_ssh_url(ssh_url: Optional[str]) -> Optional[str]:
+    """Gets regular url from ssh_url"""
+    if not ssh_url or not ssh_url.startswith("git@") or not ssh_url.endswith(".git"):
+        return ssh_url
+
+    return f"https://{ssh_url[4:-4]}"
```

Comment:
there are a lot of possible git URL formats, might be worth checking if this function can be made more generic to normalize more of them https://stackoverflow.com/q/31801271

Reply:
Ok I rewrote it to cover everything except the local filesystem transport protocol!